src/renderers/gl/renderer.cpp

```diff
@@ -319,9 +319,22 @@ mrg::Renderer::Program::Program(GLuint program_id)
     id = program_id;
     position_attr = glGetAttribLocation(id, "position");
     texcoord_attr = glGetAttribLocation(id, "texcoord");
-    tex_uniforms[0] = glGetUniformLocation(id, "tex");
-    tex_uniforms[1] = glGetUniformLocation(id, "tex1");
-    tex_uniforms[2] = glGetUniformLocation(id, "tex2");
+    for (int i = 0; i < 8 ; ++i)
+    {
+        /* You can reference uniform arrays as tex[0], tex[1], tex[2], … until you
```

Comment:
I see other code calling glGetUniformLocation(..., "tex") - I had to look that up to confirm that it still works. Maybe we should be consistent in using the index syntax?

Reply:
There is indeed other code calling `glGetUniformLocation(..., "tex")`, but they're calling it on different shader programs, which really *do* have a `tex` uniform.